docs/run-testing-framework.md

```diff
@@ -46,12 +46,12 @@ Setup only needs to be run once, it creates:
  
 Run 
 ````
-cd terraform/setup && terraform init && terraform apply
+cd terraform/setup && terraform init && terraform apply -auto-approve
```

Comment:
Should we add the auto approve. Some people might want to look at the terraform plan.

Reply:
IDC either way

Reply:
IMO, mostly the behavior from Dev when joining the first time is making sure its work beforehand. Also the frequency using terraform plan is less than terraform apply/destroy with -auto-approve. Therefore, it should be good for us to add terraform apply/destroy -auto-approve with any worries.